_posts/blog/2022-12-1-vector-library-vs-vector-database.md

```diff
@@ -108,4 +106,4 @@ If you’re interested in learning more about vector databases and vector librar
 ## Stay Connected
 Thank you so much for reading! If you would like to talk to us more about this topic, please connect with us on [Slack](https://join.slack.com/t/weaviate/shared_invite/zt-goaoifjr-o8FuVz9b1HLzhlUfyfddhw){:target="_blank"} or [Twitter](https://twitter.com/weaviate_io){:target="_blank"}. 🙂
 
-Weaviate is open-source, you can follow the project on [GitHub](https://github.com/semi-technologies/weaviate){:target="_blank"}. Don’t forget to give us a ⭐️ while you are there.
\ No newline at end of file
+Weaviate is open-source, you can follow the project on [GitHub](https://github.com/semi-technologies/weaviate){:target="_blank"}. Don’t forget to give us a ⭐️ while you are there.
```

Comment:
Can't spot the difference here.

Reply:
It was a trailing newline at the EOF. Replaced with a slightly more substantive edit.